lms/static/js/learner_dashboard/course_entitlement_factory.js

```diff
@@ -1,12 +1,7 @@
-(function(define) {
-    'use strict';
+import CourseEntitlementView from './views/course_entitlement_view';
 
-    define([
-        'js/learner_dashboard/views/course_entitlement_view'
-    ],
-    function(EntitlementView) {
-        return function(options) {
-            return new EntitlementView(options);
-        };
-    });
-}).call(this, define || RequireJS.define);
+function EntitlementFactory(options) {
+  return new CourseEntitlementView(options);
+}
+
+export { EntitlementFactory }; // eslint-disable-line import/prefer-default-export
```

Comment:
Is there a reason these aren't default exports?

Reply:
They aren't default exports because of how the static:webpack include on the html pages that consume the webpack bundles work. It inserts the namespace of the factory into window. So if we do the typical 'export default EntitlementFactory;', we end up having to reference the factory as 'window.default' in the html page.

Using a named export was my compromise there.